test/helpers/signers.js

```diff
@@ -144,4 +147,62 @@ class RSASHA256SigningKey extends RSASigningKey {
   }
 }
 
-module.exports = { NonNativeSigner, P256SigningKey, RSASigningKey, RSASHA256SigningKey };
+class MultiERC7913SigningKey {
+  #signers;
+  #weights;
+
+  constructor(signers, weights = null) {
+    assertArgument(
+      Array.isArray(signers) && signers.length > 0,
+      'signers must be a non-empty array',
+      'signers',
+      signers.length,
+    );
+
+    if (weights !== null) {
+      assertArgument(
+        Array.isArray(weights) && weights.length === signers.length,
+        'weights must be an array with the same length as signers',
+        'weights',
+        weights.length,
+      );
+    }
+
+    this.#signers = signers;
```

Comment:
We could do the sorting once at construction, and then we wouldn't have to put it in `sign()`

```suggestion
    this.#signers = signers.sort((s1, s2) => toBigInt(keccak256(signer.bytes ?? signer.address)) < toBigInt(keccak256(signer.bytes ?? signer.address)) ? -1 : 1);
```